examples/image_captioning/train.py

```diff
@@ -37,12 +37,24 @@ def main():
     parser.add_argument('--rnn', type=str, default='nsteplstm',
                         choices=['nsteplstm', 'lstm'],
                         help='Language model layer type')
-    parser.add_argument('--gpu', type=int, default=0,
-                        help='GPU ID (negative value indicates CPU)')
+    parser.add_argument('--device', '-d', type=str, default='-1',
```

Comment:
The default value looks changed. Is it intended?

Reply:
Yes. This value is used in all other examples.